olclient/openlibrary.py

```diff
@@ -126,6 +141,7 @@ def json(self):
                 """
                 exclude = ['_editions', 'olid']
                 data = { k: v for k,v in self.__dict__.items() if k not in exclude }
+                data['key'] = u'/works/' + self.olid
```

Comment:
It depends how the Work was created. Works loaded from openlibrary will have a key, but there are also the create new book methods which use `__init__` directly and don't set it. I discovered the issue while writing tests, and decided it was safer to ensure a key was always set than making sure all the test data explicitly specified a key since the OL objects would not include one by default.

I think having that shared OL object wrapper class would help handle the required keys. Currently  the Common.Book inheritance obviously doesn't require an OL key

Reply:
Ah, ok, that makes sense. Were we able to create new works in the past? If so, does this change that?